schemas/src/main/java/com/scottlogic/deg/schemas/v3/AtomicConstraintType.java

```diff
@@ -41,4 +43,10 @@
     public String toString() {
         return text;
     }
+
+    public static AtomicConstraintType fromText(String text){
```

Comment:
Can we have some tests for this method please, if nothing else to assert that the case insensitivity is working as expected.